README.md

```diff
@@ -15,7 +15,9 @@ eventdata.url = https://github.com/elastic/rally-eventdata-track
 
 ```
 
-The track can be run by specifying the following runtime parameters: `--track=eventdata` and `--track-repository=eventdata`.
+The track can be run by specifying the following runtime parameters: 
```

Comment:
Due to the way that markdown behaves the rendered output with this syntax won't look very different:

![image](https://user-images.githubusercontent.com/1754575/69408902-80d06400-0d10-11ea-8598-fcb106c12ebe.png)

I think what you are after is something like the following?

The track can be run by specifying the following runtime parameters: 

    --track=eventdata
    --track-repository=eventdata

Reply:
Yeah, that's correct.  How do you change the syntax to get it to behave that way?

Reply:
Just indent with 4 spaces in separate lines like:

>    --track=eventdata
>    --track-repository=eventdata

Reply:
Ok this should be fixed in the latest commit.